public/main/admin/user_list.php

```diff
@@ -1230,19 +1239,11 @@ class="btn btn--plain advanced_options" onclick="display_advanced_search_form();
     ['url' => api_get_path(WEB_AJAX_PATH).'usergroup.ajax.php?a=get_class_by_keyword']
 );
 
-$status_options = [];
-$status_options['%'] = get_lang('All');
-$status_options[STUDENT] = get_lang('Learner');
-$status_options[COURSEMANAGER] = get_lang('Trainer');
-$status_options[DRH] = get_lang('Human Resources Manager');
-$status_options[SESSIONADMIN] = get_lang('Course sessionsAdmin');
-$status_options[PLATFORM_ADMIN] = get_lang('Administrator');
-$status_options[STUDENT_BOSS] = get_lang('RoleStudentBoss');
-
 $form->addSelect(
-    'keyword_status',
-    get_lang('Profile'),
-    $status_options
+    'keyword_roles',
+    get_lang('Roles'),
+    array_combine(api_get_roles(), api_get_roles()),
```

Comment:
The select input should show the roles by their translated values.
The translated variables are as follow:
- ROLE_STUDENT: Learner
- ROLE_ADMIN: Admin
- ROLE_SUPER_ADMIN: Super admin
- ROLE_GLOBAL_ADMIN: Global admin
- ROLE_TEACHER: Teacher
- ROLE_HR: Human Resources Manager
- ROLE_QUESTION_MANAGER: Question manager
- ROLE_SESSION_MANAGER: Session administrator
- ROLE_STUDENT_BOSS: Superior (n+1)
- ROLE_INVITEE: Invitee

Considering we do not have such a "translations" table for roles, I would put that directly (hardcoded) into the api_get_roles() function. We will take care of that in C3.

Just a note for myself: a global admin is the person who initially installed the system (just a detail vs the super admin, which is the person managing all URLs in a multi-URL context).